SafariViewManager.m

```diff
@@ -39,48 +39,53 @@ - (void)stopObserving
         return;
     }
 
-    NSURL *url = [NSURL URLWithString:args[@"url"]];
+    NSMutableString *tempStr = [NSMutableString stringWithString:args[@"url"]];
+    [tempStr replaceOccurrencesOfString:@" " withString:@"+" options:NSCaseInsensitiveSearch range:NSMakeRange(0, [tempStr length])];
+    [tempStr replaceOccurrencesOfString:@"%09%09" withString:@"" options:NSCaseInsensitiveSearch range:NSMakeRange(0, [tempStr length])];
+    NSString *urlString = [[NSString stringWithFormat:@"%@",tempStr] stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
+    NSURL *url = [NSURL URLWithString: urlString];
```

Comment:
Maybe `[RCTConvert NSURL:args[@"url"]]` yields the desired result?

Reply:
I'll try the RCTConvert and see if that works....